lib/fetch.js

```diff
@@ -36,7 +36,7 @@ export const fetch = async (url, type, date = process.env.SCRAPE_DATE || datetim
 
   const cachedBody = await caching.getCachedFile(url, type, date);
 
-  if (cachedBody === caching.CACHE_MISS || alwaysRun) {
+  if (cachedBody === caching.CACHE_MISS || cachedBody === caching.RESOURCE_UNAVAILABLE || alwaysRun) {
```

Comment:
@qgolsteyn thoughts?

Reply:
Hmm, we might be reporting data with misleading dates with this change. Eg, if we are asking for 2020-03-18 and we don't have that in cache, we would fetch data which has been updated today 2020-03-20 and report it as if it was from 2020-03-18.

Reply:
Yeah, let's remove this change.

Reply:
Thanks. I am dropping the commit now. Let's keep in mind that without that path I could not test my patch locally with ` yarn start --date=2020-3-12 --location "DE-BY, DEU"`.

Because the fetcher returned `null` w/o even consulting the external source.

I am going to remove the commit, but leave the commit message here for posterity:
```
during testing I ran into

± yarn start --date=2020-3-12 --location "DE-BY, DEU"
yarn run v1.22.4
$ node cli.js --date=2020-3-12 --location 'DE-BY, DEU'
hourglass_flowing_sand Scraping data for 2020-3-12...
  warning Cannot go back in time to get https://covid19-germany.appspot.com/timeseries/DE-BY/cases, no cache present
  warning Cannot go back in time to get https://covid19-germany.appspot.com/timeseries/DE-BY/deaths, no cache present
  x Error processing DE-BY, DEU:  TypeError: Cannot read property 'data' of null
hourglass_flowing_sand Scraping data for 2020-2-26...
  warning Cannot go back in time to get https://covid19-germany.appspot.com/timeseries/DE-BY/cases, no cache present
  warning Cannot go back in time to get https://covid19-germany.appspot.com/timeseries/DE-BY/deaths, no cache present
  x Error processing DE-BY, DEU:  TypeError: Cannot read property 'data' of null

That means that no cache was present
and the data was still not attempted
to be fetched from the remote source.

This patch made my test command work.
```